cypress/e2e/ui-tests/test-deploy-app.cy.js

```diff
@@ -50,27 +57,27 @@ describe("Test deploying app", () => {
             cy.log("Now creating a private or project app")
             cy.get('div.card-body:contains("' + app_type + '")').find('a:contains("Create")').click()
 
-            cy.get('input[name=app_name]').type(app_name)
+            cy.get('input[name=app_name]').type(app_name_private)
             cy.get('textarea[name=app_description]').type(app_description)
             cy.get('input[name="appconfig.port"]').clear().type("8501")
             cy.get('input[name="appconfig.image"]').clear().type(image_name)
             cy.get('input[name="appconfig.path"]').clear().type("/home")
             cy.get('button').contains('Create').click()
 
             // TODO: debug problems with status not set to Running
-            //cy.get('tbody:contains("' + app_type + '")').find('span').should('contain', 'Running')
+            //cy.get('tr:contains("' + app_name_private + '")').find('span').should('contain', 'Running')
```

Comment:
I think this test i quite difficult, because it turns to running after the image has been pulled and the container has started up. This can take random amount of time. Perhaps we could check for installed or pending instead?